gtfs/gtfs_utils/gtfs_utils/config.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://example.com/config.schema.json",
+    "title": "Main configuration object parameter",
+    "description": "The configuration for a gtfs_stats run",
+    "type": "object",
+    "properties": {
+      "files": {
+        "$ref": "#/definitions/files"
+      },
+      "s3": {
+        "$ref": "#/definitions/s3"
+      },
+      "date_range": {
+        "description": "First and last date of the gtfs files to be download from bucket. {Format: *YYYY-MM-DD*}",
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^\\d{4}-\\d{2}-\\d{2}$"
+        },
+        "maxLength": 2,
+        "minLength": 2
+      },
+      "use_data_from_today": {
+        "description": "Download only gtfs data from today (overrides date_range). {default: *False*}",
+        "type": "boolean"
+      },
+      "max_gtfs_size_in_mb": {
+        "description": "Limit the maximum size of the downloaded gtfs files (in MB). If not set the limit is only free disk space.",
```

Comment:
Please add a comma after "If not set". The same for "If true, " and "If false, " in the following fields.